SUPPORT.md

```diff
@@ -1,20 +1,8 @@
-Community Supported
+# Community Supported
 
-The software and templates in the repo are released under an as-is, best effort,
-support policy. This software should be seen as community supported and Palo
-Alto Networks will contribute our expertise as and when possible. We do not
-provide technical support or help in using or troubleshooting the components of
-the project through our normal support options such as Palo Alto Networks
-support teams, or ASC (Authorized Support Centers) partners and backline support
-options. The underlying product used (the VM-Series firewall) by the scripts or
-templates are still supported, but the support is only for the product
-functionality and not for help in deploying or using the template or script
-itself. Unless explicitly tagged, all projects or work posted in our GitHub
-repository (at https://github.com/PaloAltoNetworks) or sites other than our
-official Downloads page on https://support.paloaltonetworks.com are provided
-under the best effort policy.
+This template/script/solution is released “as-is”, with no warranty and no support. These should be seen as community supported and Palo Alto Networks may contribute its expertise at its discretion. Palo Alto Networks, including through its Authorized Support Centers (ASC) partners and backline support options, will not provide technical support or help in using or troubleshooting this template/script/solution. The underlying product used by this template/script/solution will still be supported in accordance with the product’s applicable support policy and the customer’s entitlements.
```

Comment:
Sanity checks fail for the qoutes, asking for using plain ascii qoutes. I will update this. 
https://docs.ansible.com/ansible-core/devel/dev_guide/testing/sanity/no-smart-quotes.html